.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
While we're on this topic, we should add `.github`, `Makefile`, `examples`, `script` to helmignore

Reply:
oooh, great point, whoopsie